src/pages/workspaces/workspace/Data.js

```diff
@@ -947,8 +933,8 @@ const WorkspaceData = _.flow(
                         retryFunction: loadEntityMetadata,
                       },
                       [
-                        _.some({ targetWorkspace: { namespace, name } }, asyncImportJobs) && h(DataImportPlaceholder),
-                        !_.some({ targetWorkspace: { namespace, name } }, asyncImportJobs) &&
+                        runningImportJobs.length > 0 && h(DataImportPlaceholder),
+                        runningImportJobs.length === 0 &&
```

Comment:
`useImportJobs` only returns jobs for the given workspace, so there's no need to filter here.